### ISSUE TITLE
Free/Premium separation for 2.0

### ISSUE THREAD

**Issue by HardeepAsrani** [Codeinwp/otter-blocks#751]
- ~Flip is Pro (waiting for Mihai to confirm)~
- [x] Sticky can be Freemium
  - Stick to option can be free, rest for pro
- [x] WooBuilder in Pro
- [x] New Block conditions for Woo and such stay in Pro
- [x] Also Posts revamp is free but custom meta field can be in Pro

Features already in Pro: https://github.com/Codeinwp/gutenberg-blocks/issues/745 - just a note for future references.

**Comment by HardeepAsrani** [comment]
@Soare-Robert-Daniel Can you take care of Sticky and ACF option?

I'll take care of rest.

If the user is using Otter without Neve, we just show the free options.

If the user is using Otter with Neve (free), we show the free options + premium options disabled with an upsell (you can see Block Conditions for example)

And if the user is using Otter with Neve Pro with Block Editor Booster disabled, show a message to enable.

Things will get less complicated once we have the Pro version then we won't have to worry much about Neve conditions.

**Comment by Soare-Robert-Daniel** [comment]
@HardeepAsrani, @mghenciu  how the design of the message will look in this case?

Should it look like this 

![image](https://user-images.githubusercontent.com/17597852/155951807-37c8d95c-33bb-4eeb-8030-9c9fe6601525.png)

To be like links that redirect to Neve Pro

**Comment by Soare-Robert-Daniel** [comment]
For sticky:

![image](https://user-images.githubusercontent.com/17597852/155962624-7c4ce4ac-2076-4914-9d17-65f712c459e0.png)

CC: @HardeepAsrani, @mghenciu

**Comment by mghenciu** [comment]
@Soare-Robert-Daniel, as a rule of thumb, we should try to not hide Pro Features in the Free version. But keep them with a Disabled State + Upsell message. 

I think the first version is more complete. And here I think we can't show any locked meta, as those come from ACF.
The only mention would be to _**Activate Advanced Custom Fields to add more options.**_, instead of the current message. But only if adding new options requires only ACF activation. If it requires Neve Pro, we should mention that.
If Neve Pro is required, it should say: Unlock Advanced Custom Fields options, by upgrading to Neve Pro.

 For the Sticky - looks good. I would only make the the labels have a Disabled color, as the other disabled text.

**Comment by mghenciu** [comment]
Tested the build from [here](https://github.com/Codeinwp/otter-blocks/pull/689), and the message is this one - which I guess it's fine:
![image](https://user-images.githubusercontent.com/52494172/156000508-d889717d-4fe4-43d1-bc48-fa1ea264ae32.png)

For the message you mentioned above, I would choose only one action to not overwhelm users.
so it could be _**Add more data from Advance Custom Fields into your posts, by Upgrading to Neve Pro.**_
or _**Upgrade to Neve Pro and unlock more options from Advance Custom Fields**_.

basically the upgrade messages should read like one action, not many.
